src/math_util.rb

```diff
@@ -0,0 +1,43 @@
+class MathUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
```

Comment:
How can you remove the safe navigation operator `&.` from this?

Reply:
If I receive a ```nil``` parameter I need to return ```0``` because the sum is 0 or I can return ```nil``` or raise an exception.
What else can I do?